test/runtests.jl

```diff
@@ -24,6 +24,7 @@ CUDA.allowscalar() do
             include("test_stokes_drift.jl")
             include("test_utils.jl")
             include("test_schedules.jl")
+            include("test_sewater_density.jl")
```

Comment:
I don't think this belongs in unit tests because it requires a model. Perhaps `time_stepping_3`?

Reply:
Yep, moved to `time_stepping_3`